<ui version="4.0" >
 <class>WelcomeDialog</class>
 <widget class="QDialog" name="WelcomeDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Simple Chat - Выбор ника</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="sizeConstraint" >
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="margin" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Ник:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2" >
      <widget class="QLineEdit" name="nickEdit" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Настоящее имя:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2" >
      <widget class="QLineEdit" name="nameEdit" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Пол:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QComboBox" name="sexBox" />
     </item>
     <item row="2" column="2" >
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <property name="spacing" >
      <number>1</number>
     </property>
     <item>
      <widget class="QCheckBox" name="askCheckBox" >
       <property name="text" >
        <string>Больше не показывать это окно</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>ОК</string>
       </property>
       <property name="icon" >
        <iconset resource="../../data/schat.qrc" >
         <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moreButton" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../data/schat.qrc" >
         <normaloff>:/images/down.png</normaloff>:/images/down.png</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="serverLabel" >
       <property name="text" >
        <string>Адрес сервера:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serverEdit" />
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nickEdit</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>sexBox</tabstop>
  <tabstop>askCheckBox</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../data/schat.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>WelcomeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>294</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel" >
     <x>169</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moreButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>serverLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>324</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel" >
     <x>57</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moreButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>serverEdit</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>317</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel" >
     <x>309</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
